config.yaml.example

```diff
@@ -34,3 +34,4 @@ stage_logs:
     - '.sh/ir-tripleo-ffu-undercloud-rhel-upgrade.log'
     - '.sh/ir-tripleo-undercloud-rhel-upgrade.log'
     - '.sh/ir-tripleo-undercloud-ffu-upgrade.log'
+supported_versions: ['13', '16.1', '16.2']
```

Comment:
Put above the stage log stuff

Reply:
Furthermore, I feel like having this as a raw Python type input is very limiting and verbose. Why not do a more user-friendly approach?

Reply:
I like it!